modules/programs/thunderbird.nix

```diff
@@ -295,6 +317,28 @@ in {
                 argument is an automatically generated identifier.
               '';
             };
+
+            msgFilters = mkOption {
+              type = (types.listOf (types.attrs));
```

Comment:
shouldn't this be listOf $module, I guess I also like attrsOf since it can be overridden more easily, but I guess this one is ordered?

Reply:
Thunderbird requires filters to be ordered as they are applied sequentially, so listOf is necessary to preserve the correct order. For example if you are deleting certain emails no need to move them around first since they will be deleted regardless. So, filters are generally setup up like a pipeline.